02/styles/tablet.css

```diff
@@ -0,0 +1,31 @@
+body {
+    display: grid;
+
+    grid-template-columns: repeat(2, 1fr);
+}
+
+.a, .b, .c {
+    border: 3px solid transparent;
+    min-height: 50px;
+}
+
+.a {
+    border-color: magenta;
+    height: 50vh;
+    width: 100vw;
```

Comment:
lepiej na szerokość dawać 100% bo wtedy paski przewijania się nie pokazują bo 100vw + szerokość paska to więcej niż dostępna przestrzeń.